Sprint-1/exercises/random.js

```diff
@@ -7,3 +7,12 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+console.log(num);
+//1. Math.random select a random number
+//2. max value mines min and then + 1
+//3. the random number (result of step 1) multiply in result of step 2 
+//4. Math floor function select integer part of result step 3
+//5. result of step 4 sum with min number
+//6. the result assign to the num
```

Comment:
These are literal translations of the code; they may not help one understand what the code is trying to achieve.
1. What is the possible range of number return by Math.random()?
2. What is `maximum - minimum +1` trying to calculate? Why not just `maximum - minimum` or `maximum - minimum + 2`?
3. What is the possible range of numbers calculated by the expression `Math.random() * (maximum - minimum + 1)`?
4. What's the possible range of numbers returned by `Math.floor()` in line 4?

If you were asked to generate a random integer from -50 to 50. How would you write the expression?